src/extension.ts

```diff
@@ -205,6 +205,10 @@ export const activate = async (ctx: Pick<vscode.ExtensionContext, 'subscriptions
     if (activeDoc !== undefined) {
         vscode.window.showTextDocument(activeDoc)
     }
+
+    if (state.extension.packageJSON && state.extension.packageJSON.version) {
+        setTimeout(() => ui.addReplOutput(`; Alive REPL (v${state.extension.packageJSON.version})`), 250)
```

Comment:
This should be moved to the REPL view as initial content. The timeout is arbitrary, so will cause problems depending on the machine. On my slower laptop, the REPL still comes up blank.